components/timezone/src/ixdtf.rs

```diff
@@ -676,11 +684,11 @@ impl IxdtfParser {
     /// DateTime string.
     ///
     /// ```
-    /// use icu_calendar::Gregorian;
+    /// use icu_calendar::Iso;
     /// use icu_timezone::{IxdtfParser, TimeZoneInfo, UtcOffset};
     ///
     /// let tz_from_offset = IxdtfParser::new()
-    ///     .try_offset_only_from_str("2024-08-08T12:08:19-05:00", Gregorian)
+    ///     .try_offset_only_from_str("2024-08-08T12:08:19-05:00", Iso)
```

Comment:
I don't think using `Iso` as the calendar argument is a good example. First of all, you could be using `try_offset_only_from_iso_str`, and second `Iso` doesn't implement `CldrCalendar`, so it needs to be converted before formatting.

Reply:
I don't want to overuse Gregorian in docs. It is an antipattern because it is a "strong" calendar in the sense that it overrides the locale calendar during formatting if using a FixedCalendarDTF, which exists for code size only. Better style is to get the calendar from somewhere and then pass it into the parse function, not hardcode something. ISO is slightly better in this example because it can't be used for formatting. I was concerned about this in your PR earlier but didn't want to stop it from landing.

Reply:
maybe just a mix of calendars then?